app/services/process_steps.rb

```diff
@@ -1,4 +1,6 @@
 class ProcessSteps
+  attr_reader :steps
```

Comment:
Can you get rid of this service?

Reply:
Are you asking me to remove it or are you asking for my opinion? lol

Reply:
Both 😂 . 

I expected that it would get deleted, so I'm curious whether we wanted to and we couldn't (and the why), or whether we hadn't thought about it!

Reply:
Looks like not only the ProcessStepsComponent depends on it, but also VacancyFormPageHeadingComponent.

Reply:
Oh, and StepsComponent uses it too.

Reply:
The StepsComponent doesn't use it? 

I don't think the fact we use it in the heading component means that it needs to exist. A service to subtract a number seems a bit overkill. 

We could have a simple helper method instead if we didn't want to have the `current_organisation.school? ? number : number + 1` logic in a template.